src/lib/package.json

```diff
@@ -35,7 +35,7 @@
     "jquery": "^3.3.1"
   },
   "devDependencies": {
-    "@types/jasmine": "^2.8.6",
```

Comment:
Why we remove jasmine?

Reply:
`@types/jasmine` and `@types/jest` don't work well together, as `@types/jest` already contains Jasmine types.

If you don't like this, I can look for a more decent approach.

Reply:
We should support jasmine as well. If this is a problem, the consumer can install jest and his types.

Reply:
Of course. I will take a look and test it on a Jasmine-only project.